scenarios/netops/IotSensorPublishProvider/SensorPublish/publish_mqtt.py

```diff
@@ -0,0 +1,28 @@
+import time 
+import paho.mqtt.client as mqtt
+import sys
+
+topic = "ONT_DATA_SENSOR"
+broker_url = "localhost"
+broker_port = 1883
+
+
+path = sys.argv[1]
+client = mqtt.Client()
+client.connect(broker_url, broker_port, 60)
```

Comment:
why dont we just use other existing code in utils (e.g.,  similar to https://github.com/rdsea/IoTCloudSamples/blob/master/utils/plainmqttpublisher.py or a generic plainmqttpublisher) or https://github.com/rdsea/IoTCloudSamples/tree/master/IoTCloudUnits/simplesensor

It is a candidate for improvement (next update) when we do experiments?

Reply:
I will add the TODO for this potential update! Thanks for the comment